src/Workspaces/CSharp/Portable/Formatting/Rules/NewLineUserSettingFormattingRule.cs

```diff
@@ -116,6 +116,15 @@ public override AdjustSpacesOperation GetAdjustSpacesOperation(SyntaxToken previ
                 }
             }
 
+            // * { - in the anonymous Method context
```

Comment:
This comment seems wrong. Should it read `// * { - in the local function context`?